README.md

```diff
@@ -1,48 +1,45 @@
+# Compilation instructions
 
-//This is a method for background sample ntuple production on di-electron channel and e-mu channel.
-
-cmsrel CMSSW_8_0_25
+## Setup CMSSW_8_0_29
+It is required for latest JEC
 
+```bash
+cmsrel CMSSW_8_0_29
 cd CMSSW_8_0_25/src
-
 cmsenv
-
+mkdir SUSYBSMAnalysis && cd SUSYBSMAnalysis/
 git clone git@github.com:BFF-Zprime-mumu/Zprime2muAnalysis.git
```

Comment:
Must complete the electron HEEP ID setup before downloading our analysis codes.
If it does not, you get an error corresponding to the # 1 error on the slide that I announced this week.